CHANGELOG.md

```diff
@@ -54,6 +55,7 @@ Changed
 - Upgraded dependency to use boto3>=1.13.5. (#57161)
 - Changed to consistent file location handling across APIs for Juniper network devices. (#57399)
 - Use Python's hashlib (sha256) instead of shelling out (SipHash24) to generate server_id. (#57415)
+- Update `formulas.rst` with new IRC channel and links to IRC logs (#51628)
```

Comment:
I don't feel strongly that this *shouldn't* be added, but I don't feel strongly that it should be added either.

I guess I have a weak preference that it doesn't /really/ belong in the changelog, in that I would be shocked if someone was led to the correct IRC channel by encountering the entry here in the changelog.